elk/training/train.py

```diff
@@ -43,20 +44,22 @@ class RunConfig(Serializable):
     max_gpus: int = -1
     normalization: Literal["legacy", "none", "elementwise", "meanonly"] = "meanonly"
     skip_baseline: bool = False
+    concatenate_layers: int = 0
+    # if nonzero, appends the hidden states of the layer concatenate_layers before
 
 
 def train_reporter(
     cfg: RunConfig,
     dataset: DatasetDict,
     out_dir: Path,
-    layer: int,
+    layer: Union[int, list[int]],
```

Comment:
nit: we should probably just always use a list of ints, and in the case of a single layer we use a singleton list